app/graphql/mutations/create_grading.rb

```diff
@@ -23,153 +23,150 @@ class CreateGrading < ApplicationQuery
                }
              }
 
-    description 'Create grading for submission'
+    description "Create grading for submission"
 
     field :success, Boolean, null: false
 
     def resolve(_params)
       grade
       notify(
         :success,
-        I18n.t('mutations.create_grading.grade_recorded'),
-        I18n.t('mutations.create_grading.success_notification')
+        I18n.t("mutations.create_grading.grade_recorded"),
+        I18n.t("mutations.create_grading.success_notification")
       )
       { success: true }
     end
 
     class ValidateReviewData < GraphQL::Schema::Validator
       include ValidatorCombinable
+
       def validate(_object, _context, value)
-        @submission_id = value[:submission_id]
+        @error = nil
         @submission = TimelineEvent.find_by(id: value[:submission_id])
         @checklist = value[:checklist]
-        @evaluation_criteria = @submission.evaluation_criteria
+        @evaluation_criteria = @submission&.evaluation_criteria
```

Comment:
Why would evaluation_criteria criteria be empty? 🤔

Reply:
Because `submission` can be `nil`